bootstrap.sh

```diff
@@ -5,7 +5,7 @@ apt install -y python3.7 python3-pip postgresql postgresql-contrib
 sudo apt-get install libpq-dev
 pip3 install --upgrade pip
 pip3 install django==3.1
-pip3 install psycopg2
+pip3 install psycopg2 pillow
```

Comment:
Please consider using some standard method for managing Python dependencies such as `requirements.txt`, Pipenv or Poetry.

Reply:
as you suggest i create requirment.txt file and add all libarary that we need there.

Reply:
did you write the file manually, or did you generate it somehow?

Reply:
you mean like `pip3 freeze > requirements.txt`, this time i did just freeze and take the relevant part cause there is a lot of built-in stuff  that not relevant to the project, should we avoid from manually?

Reply:
unless you did not run `pip freeze` from a clean Python virtualenv - the other stuff are dependencies of your dependencies and you should freeze their versions as well.

I really recommend not using `pip` directly and using a more modern wrapper like `pip-tools` or `poetry` or `pipenv`.  But regardless of what you choose to use, you need to document the process in the README file so that other developers can do it as well.

Reply:
first, ok so i update now the requirements.txt with all the dependencies using pip3 freeze > requirements.txt. 
second- i don't understand exactly what you mean in this sentence- "you need to document the process in the README file so that other developers can do it as well."
thanks

Reply:
@rotem-benzvi there was a particular set of operations you needed to do to generate the `requirements.txt` file. A developer that would need to add a new requirements or update the existing ones will need to follow a similar process. You need to document it so they'll know what to do.

Reply:
done.